stories/module-analytics-settings.stories.js

```diff
@@ -152,8 +152,8 @@ storiesOf( 'Analytics Module/Settings', module )
 		const setupRegistry = ( { dispatch } ) => {
 			dispatch( STORE_NAME ).receiveExistingTag( null );
 			dispatch( STORE_NAME ).receiveAccounts( accounts );
-			dispatch( STORE_NAME ).receiveProperties( properties );
-			dispatch( STORE_NAME ).receiveProfiles( profiles );
+			dispatch( STORE_NAME ).receiveProperties( properties, { accountID: profiles[ 0 ].accountId } );
```

Comment:
Similar to my comment on the original PR, we should always look at the correct array here - for `receiveProperties` this should always be based on `properties`, and for `receiveProfiles` it should always be based on `profiles`. It's probably not a big deal for the data here, but this is complex enough, so I'd prefer if we used "correct" examples throughout.